extensions/positron-assistant/src/utils.ts

```diff
@@ -256,22 +256,66 @@ export function isChatImageMimeType(mimeType: string): mimeType is vscode.ChatIm
 	return Object.values(vscode.ChatImageMimeType).includes(mimeType as vscode.ChatImageMimeType);
 }
 
+export const EMPTY_TOOL_RESULT_PLACEHOLDER = 'tool result is empty';
```

Comment:
I wonder if an empty string is better, if it still fixes the issue. It'll use fewer tokens and IMO convey the same information.